### PR TITLE
DOCS: custom graphql filter fields

### PR DESCRIPTION
**Pull request by unclecheese**
Related: https://github.com/silverstripe/silverstripe-graphql/pull/431

### CODE REVIEW

File: docs/en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/02_query_plugins.md [hunk 1]
```diff
@@ -189,6 +189,82 @@ MyProject\Models\ProductCategory:
...
+
+Here's how we can resolve this custom filter:
+
+*app/src/Resolvers/Resolver.php*
+```php
+namespace MyApp\Resolvers;
+
+class Resolver
+{
+    public static function resolveHasReviewsFilter($list, $args, $context)
```
Comment (emteknetnz) [inline docs/en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/02_query_plugins.md:243]: Should put in param types in e.g. `array $context`

File: docs/en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/02_query_plugins.md [hunk 2]
```diff
@@ -189,6 +189,82 @@ MyProject\Models\ProductCategory:
...
+    {
+        $onlyWithReviews = $context['filterValue'];
+        $comparator = $context['filterComparator'];
+        
+        if (!in_array($comparator, ['eq','ne'])) {
```
Comment (emteknetnz) [inline docs/en/02_Developer_Guides/19_GraphQL/02_working_with_dataobjects/02_query_plugins.md:248]:
```suggestion
        if (!in_array($comparator, ['eq', 'ne'])) {
```

**Review by emteknetnz — CHANGES_REQUESTED** [review]

**Comment by GuySartorelli** [comment]
Note: This should not be merged here. I have started [the process of migrating docs](https://github.com/silverstripe/silverstripe-framework/issues/10215) to [a new repository](https://github.com/silverstripe/doc) so this PR should be closed and a new one should be created there.

**Comment by GuySartorelli** [comment]
Closed - docs PR opened at https://github.com/silverstripe/developer-docs/pull/59

**Pull request closed without merging**